src/libraries/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketCreationOptions.cs

```diff
@@ -52,6 +53,25 @@ public TimeSpan KeepAliveInterval
             }
         }
 
+        /// <summary>
```

Comment:
Could we update the KeepAliveInternal xml docs to mention sending a Pong when KeepAliveTimeout isn't set and a Ping when it is?

Reply:
I updated the XML comment in this file, but we will need to manually update the online docs in separate PR.